docs/src/examples/components/ItemLayout/Types/ItemLayoutExample.shorthand.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import { ItemLayout, Image } from '@stardust-ui/react'
+
+const ItemLayoutExampleShorthand = ({ knobs }) => (
+  <ItemLayout
+    media={<Image src="public/images/avatar/small/matt.jpg" avatar />}
+    header="Irving Kuhic"
+    headerMedia="7:26:56 AM"
+    content="Program the sensor to the SAS alarm through the haptic SQL card!"
+    debug={knobs.debug}
```

Comment:
Since this is the full example with a debug display knob to help the user understand the full layout,  let's also include some `endMedia` here.  Perhaps some kind of icon would be great.

Reply:
I added the end media with ellipses (&hellip;) to this examples, as well as the truncate example, as we have there even more knobs.